doc/source/tutorials/lab.rst

```diff
@@ -299,12 +342,52 @@ a two-qubit system:
                 threshold: 0.0002694329123116206
                 iq_angle: 4.912447775569025
 
+And in the case of having a chip with coupler qubits
+we need to following changes to the previous runcard:
+
+.. code-block::  yaml
+
+    qubits: [0, 1]
+    couplers: [0]
+
+    topology: {0: [0, 1]}
```

Comment:
Maybe we can add a note here that this maps coupler to qubit pair. I am not sure if it will be clear to everyone.